config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main.yaml

```
@@ -94,6 +94,8 @@ periodics:
           - runner.sh
           - "./scripts/ci-e2e.sh"
         env:
+          - name: GINKGO_ARGS
+            value: " " # set the value to empty string to override the default value.
```

Can we do
```suggestion
            value: "--fail-fast=false"
```
So we are explicit in what we want to achieve?
If not, let's write in the comment why we are overriding the default, e.g.
```suggestion
            value: " " # unset the default GINKGO_ARGS="--fail-fast" so we have a better signal by always running all the tests no matter of failures.
```